documentation/latest/gsg/ocp.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: documentation
+title: Run flotta on Openshift
+---
+
+Flotta can run on top of any Kubernetes distributions, also, main usage in
+production is on top of Openshift cluster. 
+
+To install openshift, the best way is to follow [Openshift Assisted
```

Comment:
I mentioned this in private, the main reason to get this as this, is because if someone want to try in openshift is because is already familiar with the solution, and CRC has not great dev experience at all.

Reply:
Probably a draft comment from me that shouldn't be published. I agree with your about the dev-experience with `crc`.